embedded-workshop-book/src/SUMMARY.md

```diff
@@ -26,6 +26,8 @@
       - [Starting a Project from Scratch](./from-scratch.md)
 - [Down the Stack Workbook](./down-the-stack.md)
     - [BSC Exercise](./bsc-exercise.md)
```

Comment:
This particular file `./bsc-exercise.md` shows up as empty on my version of the `mdbook` when rendering. Slight papercut, but a small blurb or image about the exercise (or an off-beat motivational sentence to encourage the participants) could occupy some space there.